python/docs/source/usage.rst

```diff
@@ -492,47 +492,77 @@ locking provider at the moment in delta-rs. To enable DynamoDB as the
 locking provider, you need to set the **AWS_S3_LOCKING_PROVIDER** to 'dynamodb'
 as a ``storage_options`` or as an environment variable.
 
-Additionally, you must create a DynamoDB table with the name ``delta_rs_lock_table``
+Additionally, you must create a DynamoDB table with the name ``delta_log``
 so that it can be automatically recognized by delta-rs. Alternatively, you can
-use a table name of your choice, but you must set the **DYNAMO_LOCK_TABLE_NAME**
+use a table name of your choice, but you must set the **DELTA_DYNAMO_TABLE_NAME**
```

Comment:
I don't recall from the pull request reviews previously, but why did this name change @dispanser ?

Reply:
In the initial implementation path, both locking solutions where applied at the same time, so the table name had to be different (due to incompatible schemas). After throwing out the old locking logic, I simply did not think about changing this.
Given that a user switching to the new library has to tear down the old lock table and set up the new one anyways, it may not be breaking any more than the logic change already does, and I do think having `DELTA` in there is actually better.

I now realize that it's not very fortunate, but with the release already made (?), changing back might make more harm than good.